samples/csharp/getting-started/DatabaseIntegration/DatabaseIntegration/DatabaseIntegration.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="EntityFramework" Version="6.2.0" />
```

Comment:
This reference shouldn't be here. I am surprised that nothing complains given that EF6.2 isn't compatible with .NET Core.